src/cat-data-service/src/service/v1/event/mod.rs

```diff
@@ -61,7 +61,15 @@ async fn events_exec(
 }
 
 /// Need to setup and run a test event db instance
-/// To do it you can use `cargo make local-event-db-test`
+/// To do it you can use the following commands:
```

Comment:
We could probably encapsulate all of these steps into a single Bash script.

Reply:
I think it will be redundant, it is needed only if you want to run it locally